### ISSUE TITLE
WCAG: Popup modals X icon

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2206]
ll pop-up screens on this website have a cross icon in the top right corner to close the screen. These buttons that act as buttons are made from non-interactive div elements and lack all the features of a button. This success criterion includes keyboard operation.

These buttons are not accessible to people without a mouse.

Make sure that these buttons can also be operated with the keyboard. (ESC button for example)
